src/status_im2/navigation/screens.cljs

```diff
@@ -85,10 +86,12 @@
      :component how-to-pair/instructions}
 
     {:name      :discover-communities
-     :component communities.discover/discover}
+     :component communities.discover/discover
+     :options   {:layout {:orientation ["portrait"]}}}
```

Comment:
i'm not sure why its needed here, because we didn't have options here before, so default options should worked here